Message/Common/UserRepository.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UserRepository.swift
+//  Message
+//
+//  Created by Minh Tâm on 11/27/19.
+//  Copyright © 2019 Minh Tâm. All rights reserved.
+//
+
+import Foundation
+import FirebaseAuth
+import UIKit
+
+class UserRepository {
+    
+    let defaults = UserDefaults.standard
+    
+    func login(user : String , password : String , completion : ((String?, Error?) -> Void)?) {
+        FireStoreManager.shared.login(user: user, password: password) {_, error -> Void in
+            if error != nil {
+                ViewController.shared.showAlert(message: "Đã có lỗi xảy ra", title: "Thử lại")
```

Comment:
```suggestion
                completion("Error", error)
```
Không đẩy ViewController xuống Repository

Reply:
Nó bị thế này anh ưi
<img width="854" alt="Screen Shot 2019-11-28 at 11 06 24" src="https://user-images.githubusercontent.com/35531609/69776541-457ddb80-11cf-11ea-8093-1d5dbf8e03f0.png">

Reply:
@minhtamkv em sửa thành 
```swift
completion?("Error", error)
``` 
là ok nhé